ios/TdReactNativeSdk.m

```diff
@@ -9,9 +9,8 @@ @implementation TdReactNativeSdk
 
 RCT_EXPORT_METHOD(setup:(NSDictionary *)configuration)
 {
-    [TreasureData initializeApiEndpoint:configuration[@"apiEndpoint"]];
     [TreasureData initializeEncryptionKey:configuration[@"encryptionKey"]];
-    [TreasureData initializeWithApiKey:configuration[@"apiKey"]];
+    [TreasureData initializeWithApiKey:configuration[@"apiKey"] apiEndpoint:configuration[@"apiEndpoint"]];
```

Comment:
```suggestion
    if (configuration[@"apiEndpoint"]) {
        [TreasureData initializeWithApiKey:configuration[@"apiKey"] apiEndpoint:configuration[@"apiEndpoint"]];
    } else {
         [TreasureData initializeWithApiKey:configuration[@"apiKey"]];
    }
```

You will have to include an if condition here. In `td-ios-sdk` apiEndpoint is _Nonnull.